src/runtime/env.rs

```diff
@@ -368,3 +374,34 @@ fn migrate_storage_schema_to_v5<E: Env>() -> TryEnvFuture<()> {
         .and_then(|_| E::set_storage(SCHEMA_VERSION_STORAGE_KEY, Some(&5)))
         .boxed_env()
 }
+
+fn migrate_storage_schema_to_v6<E: Env>() -> TryEnvFuture<()> {
+    E::get_storage::<serde_json::Value>(PROFILE_STORAGE_KEY)
+        .and_then(|mut profile| {
+            match profile
+                .as_mut()
+                .and_then(|profile| profile.as_object_mut())
+                .and_then(|profile| profile.get_mut("settings"))
+                .and_then(|settings| settings.as_object_mut())
+            {
+                Some(settings) => {
+                    settings.insert(
+                        "player_type".to_owned(),
+                        serde_json::Value::Number(serde_json::Number::from(0)),
```

Comment:
player_type is declared as Option<String>
but here we put 0
I suggest we use serde_json::Value::NULL

Reply:
Actually, we can just remove it, since it should deserialize to `None` if there's nothing there (same was done with secondary languages settings). Tho maybe it would be good to check whether the `play_in_external_player` flag is set and store `external` in the `player_type`, tho not entirely sure how to impl this.

Reply:
i thought we should use player_type only for the internal player preference
if we use it for external too maybe we can combine those 2 settings

Reply:
Yeah, I think it makes sense to just have one setting, since you can only use one player, so `external` is also a sort of player in that sense.

Reply:
yes, so the migration should also delete `play_in_external` bool and insert `external` if that flag was previously set to true.
merging of this pr will then require changes in the apps.
we can keep the checkbox for 'play in external player' but instead of setting the flag to true we can set that string to `external`

Reply:
Yes, makes sense to remove it. I've added a migration for it, but would be good if you could check it if everything's correct.